tests/rest_tec_v1_integration/Endpoints/Ticket_Test.php

```diff
@@ -75,37 +75,45 @@ protected function create_test_data(): array {
 		$ticket_1 = $this->create_tc_ticket( $post_1, '25.00' );
 		update_post_meta( $ticket_1, '_name', 'General Admission' );
 		update_post_meta( $ticket_1, '_description', 'Standard ticket for general admission' );
+		wp_update_post( [ 'ID' => $ticket_1, 'menu_order' => 1 ] );
 
 		$ticket_2 = $this->create_tc_ticket( $post_1, '75.00' );
 		update_post_meta( $ticket_2, '_name', 'VIP Ticket' );
 		update_post_meta( $ticket_2, '_description', 'VIP access with special perks' );
+		wp_update_post( [ 'ID' => $ticket_2, 'menu_order' => 2 ] );
 
 		$ticket_3 = $this->create_tc_ticket( $page_1, '15.00' );
 		update_post_meta( $ticket_3, '_name', 'Workshop Registration' );
 		update_post_meta( $ticket_3, '_description', 'Basic workshop registration' );
+		wp_update_post( [ 'ID' => $ticket_3, 'menu_order' => 3 ] );
 
 		// Create ticket for private post
 		$private_ticket = $this->create_tc_ticket( $private_post, '50.00' );
 		update_post_meta( $private_ticket, '_name', 'Private Event Ticket' );
+		wp_update_post( [ 'ID' => $private_ticket, 'menu_order' => 4 ] );
```

Comment:
Since the post ID is always part of the natural ordering, having a `menu_order` replicate the natural post ID ordering is not telling you whether the `menu_order` based-ordering is working or not. I would assign, with a comment explaining why, a `menu_order` that does not follow the natural post ID ordering.
E.g. th first ticket created ha a `menu_order` of `2`, the second `1` and so on.